cfgov/v1/atomic_elements/organisms.py

```diff
@@ -611,6 +611,12 @@ class FilterControls(BaseExpandable):
                                      label='Filter Date Range')
     output_5050 = blocks.BooleanBlock(default=False, required=False,
                                       label="Render preview items as 50-50s")
+    should_link_image = blocks.BooleanBlock(
+        default=False,
+        required=False,
+        help_text=('Check this to link all images to the URL of the first '
+                   'link in their unit\'s list, if there is a link.')
```

Comment:
This text was a little bit confusing at first. What about something like "Add links to post preview images in filterable list results".